archetypes/pattern/_index.adoc

```diff
@@ -13,7 +13,7 @@ industries:
 links:
   install:
   arch:
-  help: https://groups.google.com/g/hybrid-cloud-patterns
```

Comment:
I took the liberty of fixing those via https://github.com/validatedpatterns/docs/pull/324 since the old group was already returning "content unavailable"